createNPMPackage.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+yarn install --immutable
+
+if [ $# -ge 1 ]; then
+  if ! CURRENT_VERSION=$(node scripts/set-screens-version.js "$@"); then
+    exit 1
+  fi
+fi
+
+yarn build
+
+npm pack
+
+if [ $# -ge 1 ]; then
+  node scripts/set-screens-version.js "$CURRENT_VERSION" >/dev/null
```

Comment:
I don't think we need to include package name in the script name
```suggestion
  node scripts/set-version.js "$CURRENT_VERSION" >/dev/null
```